### PR TITLE
Docs: fix grammar for couple of rules

### LINKED ISSUE #158: Fix grammar in a couple of rules descriptions
According to @hzoo notes in PR: https://github.com/jscs-dev/node-jscs/pull/1803

### PR DESCRIPTION
**Pull request by zxqfox**
Fixes #158

/cc @hzoo @markelog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,13 +71,15 @@ To use plugin you should add these lines to configuration file `.jscsrc`:
 
 ### checkAnnotations
 
-Ensures tag names are valid
+Checks tag names are valid
 
 There are 3 presets for `Closure Compiler`, `JSDoc3` and `JSDuck5`.
 
-By default it allows any tag of mixed set. You can pass `Object` to select preset with `preset` field and add custom tags with `extra` field.
+By default it allows any tag of mixed set. You can pass `Object`
```
Comment (hzoo) [inline README.md]: What did we say about this again? `By default it allows any tag of mixed set` I don't think mixed is that clear - maybe something like `any tag from the set of presets` or `any tag from the 3 presets`?

File: README.md [hunk 2]
```diff
@@ -150,7 +152,7 @@ function _f() {}
 
 ### checkParamExistence
 
-Ensures all parameters are documented.
+Checks all parameters are documented.
```
Comment (hzoo) [inline README.md]: We should decide if these should all have periods or not for consistency

File: README.md (hunk 1 above)
Comment (zxqfox) [inline README.md]: `any tag from any preset`? `any known tag`?

File: README.md (hunk 2 above)
Comment (zxqfox) [inline README.md]: Guess there was a rule for it ;-)

File: README.md (hunk 1 above)
Comment (hzoo) [inline README.md]: Ok `any tag from any preset`

**Comment by hzoo** [comment]
:+1: since most of it is changing to `Checks` - will probably have to look through it again later

**Pull request closed without merging**